doc/source/api/solver/fieldinfo.rst

```diff
@@ -0,0 +1,98 @@
+.. _ref_field_info:
+
+Field info
+==========
+
+You can use field info objects to access Fluent field information.
+
+Accessing field info objects
+----------------------------
+
+In order to access field info, launch the fluent solver, and make field
+data available (for example, by reading case and data files):
+
+.. code-block:: python
+
+  >>> import ansys.fluent.core as pyfluent
+  >>> solver = pyfluent.launch_fluent(mode="solver")
+  >>> solver.file.read(file_type="case-dats", file_name=mixing_elbow_case_path)
+  >>> solver.solution.initialization.hybrid_initialize()
+
+
+The field info object is an attribute of the solver object:
+
+.. code-block:: python
+
+  >>> field_info = solver.field_info
+
+Sample requests
+---------------
+
+Here are the methods for requesting field information:
+
+- ``get_fields_info`` for getting fields information.
+- ``get_range`` for getting the range of the field.
+- ``get_vector_fields_info`` for getting vector fields information.
+- ``get_surfaces_info`` for getting the surfaces information.
+
+Get fields info
+~~~~~~~~~~~~~~~
+You can request the fields information (field name, domain, and section) by
+calling the ``get_fields_info`` method.
+
+.. code-block:: python
+
+  >>> field_info.get_fields_info()
+  {'pressure': {'display_name': 'Static Pressure', 'section': 'Pressure...', 'domain': 'mixture'},
+   'pressure-coefficient': {'display_name': 'Pressure Coefficient', 'section': 'Pressure...', 'domain': 'mixture'},
+   'dynamic-pressure': {'display_name': 'Dynamic Pressure', 'section': 'Pressure...', 'domain': 'mixture'},
+   'absolute-pressure': {'display_name': 'Absolute Pressure', 'section': 'Pressure...', 'domain': 'mixture'},
+    ...}
+
+Get range
+~~~~~~~~~
+You can request the range (minimum and maximum values) for a given ``field`` by
+calling the ``get_range`` method. It takes a ``field`` argument which can be obtained
+from the keys of the dictionary returned by ``get_fields_info`` method.
+
+.. code-block:: python
+
+  >>> field_info.get_range("velocity")
+  [0.0, 0.0]
+
+Get vector fields info
+~~~~~~~~~~~~~~~~~~~~~~
+You can request the vector fields information by calling the
+``get_vector_fields_info`` method.
+
+.. code-block:: python
+
+  >>> field_info.get_vector_fields_info()
+  {'velocity': {'x-component': 'x-velocity', 'y-component': 'y-velocity', 'z-component': 'z-velocity'},
+   'relative-velocity': {'x-component': 'relative-x-velocity', 'y-component': 'relative-y-velocity', 'z-component': 'relative-z-velocity'}}
+
+Get surfaces info
+~~~~~~~~~~~~~~~~~
+You can request the surfaces information (surface name, ID, and type) by
+calling the ``get_surfaces_info`` method.
+
+.. code-block:: python
+
+  >>> field_info.get_surfaces_info()
+  {'symmetry-xyplane': {'surface_id': [5], 'zone_id': 29, 'zone_type': 'symmetry', 'type': 'zone-surf'},
```

Comment:
Same here... this interface could return a list of SurfaceInfo objects.

Reply:
#1285